lib/pushmi_pullyu/cli.rb

```diff
@@ -89,6 +95,20 @@ def parse_options(argv)
         config.monitor = true
       end
 
+      opts.on('-rh', '--redis-host IP', 'Host IP of Redis instance to read from.'\
```

Comment:
Created issue for above commet here: https://github.com/ualbertalib/pushmi_pullyu/issues/47